jbi/services/jira.py

```diff
@@ -124,16 +136,24 @@ def check_health(self, actions: Actions) -> ServiceHealth:
         health: ServiceHealth = {
             "up": is_up,
             "all_projects_are_visible": is_up and self._all_projects_visible(actions),
-            "all_projects_have_permissions": self._all_projects_permissions(actions),
-            "all_projects_components_exist": is_up
-            and self._all_projects_components_exist(actions),
+            "all_projects_have_permissions": is_up
+            and self._all_projects_permissions(actions),
+            "all_project_custom_components_exist": is_up
+            and self._all_project_custom_components_exist(actions),
             "all_projects_issue_types_exist": is_up
             and self._all_project_issue_types_exist(actions),
         }
         return health
 
     def _all_projects_visible(self, actions: Actions) -> bool:
-        visible_projects = {project["key"] for project in self.fetch_visible_projects()}
+        try:
+            visible_projects = {
+                project["key"] for project in self.fetch_visible_projects()
+            }
+        except requests.HTTPError:
+            logger.exception("Error fetching visible Jira projects")
```

Comment:
Is it worth logging the exception thrown to be verbose?

Reply:
Python does this automagically when you do [`logger.exception`](https://docs.python.org/3/library/logging.html#logging.Logger.exception), but good call

Reply:
Well then, TIL